### PR TITLE
Sample Function App for .NET 4.6

### PR DESCRIPTION
**Pull request by issacnitin**

### CODE REVIEW

**Comment by sachinma** [comment]
Can you follow the naming conventions followed in this repo.
Name the top folder, functionApp
Put the code inside Application
And also have ArmTemplates folder

**Comment by issacnitin** [comment]
@sachinma  I've addressed the comment. Thanks!

File: dotnet/aspnet46/functionApp/ArmTemplates/windows-functionapp-template.json [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+                            "enabled": true
+                        }
+                    },
+                    "keySource": "Microsoft.Storage"
+                }
+            },
+            "dependsOn": []
+        },
+      {
+        "type": "Microsoft.Web/serverfarms",
```
Comment (sachinma) [inline dotnet/aspnet46/functionApp/ArmTemplates/windows-functionapp-template.json]: Is this the template for the consumption based hosted plan?

Comment (issacnitin) [reply dotnet/aspnet46/functionApp/ArmTemplates/windows-functionapp-template.json]: Yes, the default is the consumption-based plan(Y1  Dynamic).

**Pull request merged**